nav2_dwb_controller/dwb_plugins/src/standard_traj_generator.cpp

```diff
@@ -88,18 +87,6 @@ void StandardTrajectoryGenerator::initializeIterator(
   velocity_iterator_->initialize(nh, kinematics_);
 }
 
-void StandardTrajectoryGenerator::checkUseDwaParam(
```

Comment:
Please add to the DWB readme that they should use the `LimitedAccelGenerator` for DWA behavior and `StandardTrajectoryGenerator` for DWB behavior